spec/models/category_spec.rb

```diff
@@ -1,10 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe Category, :type => :model do
+  context "Create a new category" do
+    let(:category) do
+      FactoryGirl.create(
+        :category,
+        name: "Test Category",
+        parent_id: 30
+      )
+    end
 
+    it "To have a name." do
+      expect(category.name).to eq("Test Category")
+    end
 
-  context 'moving parent category up or down once: ' do
+    it "To have a display index." do
+      expect(category.display_index).to eq(1)
+    end
 
+    it "To belong to a parent category." do
+      expect(category.parent_id).to eq(30)
+    end
+
+    it "To belong to a user." do
+      expect(category.user).to_not eq(nil)
+    end
+  end
+
+  context 'moving parent category up or down once: ' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.